packages/react-router/docs/api/Route.md

```diff
@@ -178,3 +178,11 @@ When `true`, a `path` that has a trailing slash will only match a `location.path
 | `/one` | `/one` | yes |
 | `/one` | `/one/` | no |
 | `/one` | `/one/two` | no |
+
+## location: object
+
+A `<Route>` element tries to match its `path` to the current browser `location`.
```

Comment:
It should be "the current *history* `location`", since it doesn't always have to be in a browser.